pkg/component/interop_test.go

```diff
@@ -158,6 +164,6 @@ func TestInteropTLS(t *testing.T) {
 		if !a.So(json.NewDecoder(res.Body).Decode(&msg), should.BeNil) {
 			t.FailNow()
 		}
-		a.So(msg.Result, should.Equal, interop.ResultUnknownSender)
+		a.So(msg.Result.ResultCode, should.Equal, interop.ResultUnknownSender)
```

Comment:
I'd prefer asserting `msg.Result` or even the whole `msg` via `should.Resemble` to ensure nothing is missed, but that's personal preference I guess.